src/machinetalk/messagebus/messagebus.cc

```diff
@@ -411,7 +411,7 @@ static int rtproxy_setup(msgbusd_self_t *self)
     demo.from_rt_name = "mptx.0.out";
     demo.min_delay = 2;   // msec
     demo.max_delay = 200; // msec
-    demo.pipe = zthread_fork (self->context, rtproxy_thread, &demo);
+    demo.pipe = zactor_fork (self->context, rtproxy_thread, &demo);
```

Comment:
There does not appear to be a method `zactor_fork()`

The code will compile however, same goes for many other calls which don't seem to exist, but do not throw an error :(